maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/index.ts

```diff
@@ -384,36 +397,56 @@ export class Project {
 
 			return autoPresets;
 		}, []);
-	}
 
-	_loadDefaultPreset(): string {
-		let presetFilePath;
-
-		const autoPresets = this._findAutopresets();
-
-		if (autoPresets.length > 1) {
+		if (autopresets.length > 1) {
 			throw new Error(
 				'Multiple autopreset dependencies found in project (' +
-					autoPresets +
+					autopresets +
 					'): please remove the invalid ones or ' +
 					'explicitly define the preset to be used in the ' +
 					'.npmbundlerrc file'
```

Comment:
What's the reason for us not using template literals here? We're using it below on line 437

Reply:
Probably none :sweat_smile: . Let me review and change it.

Reply:
Addressed in https://github.com/liferay/liferay-frontend-projects/pull/576